### PR TITLE
Improve UI

### PR DESCRIPTION
**Pull request by kulisic4**
Improving UI, completely redesigned Home and Blog feature.

### CODE REVIEW

File: component/Counter/index.tsx [hunk 1]
```diff
@@ -1,37 +1,37 @@
...
+	const handleIncrement = () => {
+		setCount(count + 1);
+	};
 
-  const handleDecrement = () => {
-    setCount(count - 1);
-  };
+	const handleDecrement = () => {
+		setCount(count - 1);
+	};
```
Comment (nikolaklenje) [inline component/Counter/index.tsx:12]: For smaller functions that you not reuse in other parts of code you can always write them inline (inside of the tag 
and have fewer lines of code)

File: component/Fact/index.tsx [hunk 2]
```diff
@@ -24,14 +24,14 @@ export const Fact: FC<any> = ({ fact, setFacts }: any) => {
...
 			<p>
 				{badSource ? (
 					<span className="px-2 py-2">[ ⛔️ BAD SOURCE ]</span>
 				) : null}
 				{fact.text}
-				<a className="source" href={fact.source} target="_blank">
-            (Source)
+				<a className="source italic" href={fact.source} target="_blank">
+					{" "}(Source)
 				</a>
```
Comment (nikolaklenje) [inline component/Fact/index.tsx:35]: Use Link from NextJs and find out what are advantages of using it.

File: component/Post/index.tsx [hunk 3]
```diff
@@ -1,51 +1,70 @@
...
+				const posts = response?.data?.data?.posts;
+				if (posts) {
+					setBlogPosts(posts);
+				}
+			})
+			.catch((error) => {
+				console.error("Error fetching blog posts:", error);
+			});
+		console.log("BlogContent");
+	}, []);
```
Comment (nikolaklenje) [inline component/Post/index.tsx:22]: Do not use empty dependency in useEffect. Find another way

File: component/PostCard/index.tsx [hunk 4]
```diff
@@ -1,74 +1,162 @@
...
+				const posts = response?.data?.data?.posts;
+				if (posts) {
+					setBlogPosts(posts);
+				}
+			})
+			.catch((error) => {
+				console.error("Error fetching blog posts:", error);
+			});
+		console.log("BlogPost");
+	}, []);
```
Comment (nikolaklenje) [inline component/PostCard/index.tsx:28]: Do not use empty dependency

File: component/PostCard/index.tsx [hunk 5]
```diff
@@ -1,74 +1,162 @@
...
+			return true;
+		}
 
+		if (
+			post.categories.some((cat) => cat.name === selectedCategory) &&
+      post.author.name === selectedAuthor
+		) {
+			// Show posts that match both selected category and author
+			return true;
+		}
```
Comment (nikolaklenje) [inline component/PostCard/index.tsx:81]: Too many repetitions

File: component/StoreItems/index.tsx [hunk 6]
```diff
@@ -61,74 +54,60 @@ export const StoreItems: FC = () => {
...
-            </h1>
-          </div>
-          <div className="grid grid-flow-col auto mt-10-cols-auto mt-10 ml-5 md:ml-24 py-16 pt-20 pt-20 pb-0 store-items">
-            <div className="bg-teal-100 rounded-l-lg border-solid border-2 border-teal-500 ... shadow-2xl hover:bg-teal-200 text-xl font-serif">
-              <p className="px-5 md:px-20 py-5 md:py-20">
+						</h1>
+					</div>
+					<div className="grid grid-flow-col auto mt-10-cols-auto mt-10 ml-5 py-16 pt-20 pb-0 store-items">
+						<div className="bg-white rounded-lg border-solid border-2 border-teal-500 ... shadow-2x. text-xl font-serif mr-5">
+							<p className="px-5 md:px-20 py-5 md:py-20">
```
Comment (nikolaklenje) [inline component/StoreItems/index.tsx:79]: Again white spaces

File: component/StoreItems/index.tsx [hunk 7]
```diff
@@ -61,74 +54,60 @@ export const StoreItems: FC = () => {
...
-              </div>
-            </div>
-          </div>
-          <div className="grid grid-flow-col auto mt-10-cols-auto arrow-block">
-            <button className="hover:bg-teal-100 bg-teal-500 text-white hover:text-red-500 font-bold py-2 px-4 ml-5 md:ml-80 rounded-lg rounded-l-lg border-solid border-2 border-teal-500 ... shadow-2xl hover:bg-teal-200 text-xl font-serif w-40 md:w-60 mt-10">
-              <Link href="https://eatsomos.com/products/smoky-chipotle-mushrooms-entree-2-pack?sscid=71k7_ztiqn&">
+							</p>
+						</div>
+						<div className="mr-6">
+							<img className="h-full border-2 border-teal-500 rounded-lg" src="https://cdn.outsideonline.com/wp-content/uploads/2022/10/somos-smoky-chipotle_h.jpg?width=730" />
```
Comment (nikolaklenje) [inline component/StoreItems/index.tsx:90]: Again Image

File: lib/constants/index.ts [hunk 8]
```diff
@@ -5,16 +5,16 @@ interface Categories {
...
+	{ name: "Mental_Health", color: "#ffffff" },
+	{ name: "Habit_Development", color: "#ffffff" },
+	{ name: "Brain_Food", color: "#ffffff" },
+	{ name: "Science", color: "#ffffff" },
+	{ name: "Strength", color: "#ffffff" },
+	{ name: "Endurance", color: "#ffffff" },
+	{ name: "Anxiety", color: "#ffffff" },
+	{ name: "Workout", color: "#ffffff" },
+	{ name: "News", color: "#ffffff" },
 ];
```
Comment (nikolaklenje) [inline lib/constants/index.ts:20]: Too many repetitions.

**Review by nikolaklenje — CHANGES_REQUESTED** [review]

**Review by slavo3dev — APPROVED** [review]

**Pull request merged**